### PR TITLE
RE: Fix UI Layout

### PR DESCRIPTION
**Pull request by Adyel**
- Fixed conflict
- Fixed code formatting
- Added UI Layout

### CODE REVIEW

**Comment by Adyel** [comment]
@mominul  Please do a final check before pushing to the main branch.

File: src/frontend/SettingsDialog.ui [hunk 1]
```diff
@@ -1,178 +1,133 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ui version="4.0">
```
Comment (mominul) [inline src/frontend/SettingsDialog.ui]: settings_cutoff
The text is being cut off.

File: src/frontend/AboutDialog.ui [hunk 2]
```diff
@@ -1,869 +1,2568 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (mominul) [inline src/frontend/AboutDialog.ui]: too_much_about
Too much spaces!

File: src/frontend/AboutFile.ui [hunk 3]
```diff
@@ -6,153 +6,152 @@
    <rect>
```
Comment (mominul) [inline src/frontend/AboutFile.ui]: about_layout_mistakes
That `Ok` button is too much wide :)

File: src/frontend/AutoCorrectDialog.ui [hunk 4]
```diff
@@ -1,228 +1,167 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (mominul) [inline src/frontend/AutoCorrectDialog.ui]: autocorrect_mistakes
That two text boxes are unequal in size. `Clear` button is also quite inconsistent.

File: src/frontend/AboutDialog.ui (hunk 2 above)
Comment (Adyel) [reply src/frontend/AboutDialog.ui]: This is OLD School one, I'm not gonna change it's layout for now. It requires some rework.

**Comment by Adyel** [comment]
@mominul 
Fixed the look

**AboutFile.ui**
image


**SettingsDialog.ui**
This looked fine on my End. After your report I added padding just in case. Please, recheck if it is fixed.
image

**AutoCorrectDialog.ui**
image

**Comment by Adyel** [comment]
@mominul  So, looks ok? Merge?

**Comment by mominul** [comment]
Sorry I haven't tested out the changes yet! 😫

File: src/frontend/SettingsDialog.ui [hunk 5]
```diff
@@ -13,132 +13,139 @@
   <property name="windowTitle">
```
Comment (mominul) [inline src/frontend/SettingsDialog.ui]: settings_flaw
The padding didn't work. There are now two lines which are being cut off.

Comment (Adyel) [reply src/frontend/SettingsDialog.ui]: Hmm, That's weird. This window supposes to have a fixed width. It is showing way smaller in your screenshot.

Comment (Adyel) [reply src/frontend/SettingsDialog.ui]: @mominul Can you come to discord.

**Comment by mominul** [comment]
Merging this now. Will have further discussions at #56 . Thanks for the work @Adyel !

**Pull request merged**